hw/ip/rstmgr/data/rstmgr_sec_cm_testplan.hjson

```diff
@@ -27,43 +27,55 @@
       name: sec_cm_bus_integrity
       desc: "Verify the countermeasure(s) BUS.INTEGRITY."
       milestone: V2S
-      tests: []
+      tests: ["rstmgr_tl_intg_err"]
     }
     {
       name: sec_cm_scan_intersig_mubi
       desc: "Verify the countermeasure(s) SCAN.INTERSIG.MUBI."
       milestone: V2S
-      tests: []
+      tests: ["rstmgr_sec_cm_scan_intersig_mubi"]
     }
     {
       name: sec_cm_leaf_rst_bkgn_chk
-      desc: "Verify the countermeasure(s) LEAF.RST.BKGN_CHK."
+      desc: "Verify the countermeasure(s) LEAF.RST.BKGN_CHK. XXX need more info"
```

Comment:
perhaps add a TODO. And after update it later after get clarification from designer.